.github/pull_request_template.md

```diff
@@ -30,13 +30,19 @@
 - [ ] 202006
 - [ ] 202012
 - [ ] 202106
+- [ ] 202111
 
 #### Description for the changelog
 <!--
 Write a short (one line) summary that describes the changes in this
 pull request for inclusion in the changelog:
 -->
 
+#### Link to config_db schema for YANG module changes
```

Comment:
Thanks for this PR. We can also add Page details.

Provide a link to config_db schema for the table for which YANG model
is defined.
Link should point to correct section on https://github.com/Azure/SONiC/wiki/Configuration.

Reply:
Done